contracts/HubPool.sol

```diff
@@ -26,6 +26,13 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 
+/**
+ * @notice Contract deployed on Ethereum that houses L1 token liquidity for all SpokePools. Admin can instruct this
+ * contract to send tokens to L2 SpokePools via "pool rebalances" with which to pay out relayers on those networks.
+ * Admin also can publish relayer refund and slow relay merkle roots to SpokePools via this contract.
```

Comment:
Again, not sure if we should use the Admin word. wdyt?

Reply:
sure changing